tests/test_ijit/test_ijit_without_numba.py

```diff
@@ -19,5 +19,25 @@ def _simple_func(a, b):
         assert len(w) >= 1
         assert isinstance(_simple_func, types.FunctionType)
 
+@mock.patch.dict(sys.modules, {"numba": None})
+def test_function_given():
+    with warnings.catch_warnings(record=True) as w:
+        custom_jit = reload(einsteinpy.ijit)
+
+        def _simple_func(a, b):
+            return a + b
 
-custom_jit = reload(einsteinpy.ijit)
+        return_func = custom_jit.jit(_simple_func)
+
+        assert len(w) >= 1
+        assert return_func.__name__ == "_simple_func"
+
+
+@mock.patch.dict(sys.modules, {"numba": None})
+def test_no_function_given():
```

Comment:
`assert res.__name__ == "_jit"` in my suggestion above covers this test. You can remove this function.